Sources/Components/Components/Watchlist/WKWatchlistView.swift

```diff
@@ -103,15 +113,20 @@ private struct WKWatchlistViewCell: View {
 									.frame(maxWidth: .infinity, alignment: .topLeading)
 							}
 
-							// TODO: Replace with user button
-							Menu(itemViewModel.username) {
-								Button("User page", action: { })
-								Button("User talk page", action: { })
-								Button("User contributions", action: { })
-								Button("Thank", action: { })
+							HStack {
+								WKSwiftUIMenuButton(configuration: WKMenuButton.Configuration(
+									title: itemViewModel.username,
+									image: UIImage(systemName: "person.fill"),
```

Comment:
Do you think this should flow through `WKSFSymbolIcon`? I think it could be helpful in the far future to have it serve as an overview of all the symbols we're using in the app, plus you can bury renderingMode in there and delete [this line](https://github.com/wikimedia/wikipedia-ios-components/pull/25/files#diff-25d44a8ef9abfd09e0478bfcf6b19fd6146a6f1cf3469d977a3f2cd41ea4309cR32).

Reply:
Good catch - updated, thanks! Also refactored my added symbols to match the naming pattern in there, and removed one of my extraneously added symbols to reuse an existing one.